### ISSUE TITLE
Plushy background is on both sites right now. Need to change OKC's. 

### ISSUE THREAD

**Issue by BenhamL** [oakwoodgates/happyhour#33]

**Comment by BenhamL** [comment]
@oakwoodgates did you see this one?

**Comment by oakwoodgates** [comment]
Yes, we're switching the purple/pink for tulsa, and the background for okc...is that right?

**Comment by oakwoodgates** [comment]
which background?
http://45.55.219.172/wp-content/themes/happyhour/assets/img/background/8.jpg
http://content.wallpapers-room.com/resolutions/1920x1200/B/Wallpapers-room_com___Black_Background_Set_Leather_by_iAmFreeman_1920x1200.jpg
